WcaOnRails/app/models/user.rb

```diff
@@ -329,6 +329,10 @@ def can_manage_competition?(competition)
     can_admin_results? || competition.organizers.include?(self) || competition.delegates.include?(self)
   end
 
+  def can_edit_registration?(registration)
```

Comment:
Note the `can_manage_competition?` right before this. If this method should go on Registration, then `can_manage_competition?` should probably also go on Competition. I personally like how all our abilities are together right here in this one file, but maybe that doesn't scale forevery.